src/layouts/admin/sidebar-sections.tsx

```diff
@@ -1,139 +1,167 @@
-import {
-  Abc,
-  AddBox,
-  AllInclusive,
-  Book,
-  Dashboard,
-  Description,
-  Extension,
-  GolfCourse,
-  Group,
-  Hardware,
-  LibraryBooks,
-  Pin,
-  ViewInAr,
-} from "@mui/icons-material";
+import { Icon } from "@iconify/react";
 
 export const sidebarSections = [
   {
     name: "Courses",
-    icon: <GolfCourse />,
+    icon: <Icon icon="mdi:golf" width="32" height="32" />,
```

Comment:
here i would just store the string, and then map that value into the component so you can convert this file to .ts

Reply:
will sort